force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,8 +32,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
-    }
+        if(a != null && b != null){
```

Comment:
Its typically best practice to handle null values first, then proceed to processing any logic once you establish the parameters have values to work with.